src/default_panels/StyleColorbarsPanel.js

```diff
@@ -185,49 +186,54 @@ const StyleColorBarsPanel = (props, {localize: _}) => {
                   {label: _('k/M/B'), value: 'B'},
                 ]}
               />
-              <PlotlySection name={_('Label Prefix')}>
-                <Dropdown
-                  label={_('Prefix')}
-                  attr={prefix + 'colorbar.tickprefix'}
-                  options={[
-                    {label: _('x'), value: 'x'},
-                    {label: _('$'), value: '$'},
-                    {label: _('#'), value: '#'},
-                    {label: _('@'), value: '@'},
-                    {label: _('custom'), value: 'custom'},
-                  ]}
-                />
-                <Radio
-                  attr={prefix + 'colorbar.showtickprefix'}
-                  options={[
-                    {label: _('Every'), value: 'all'},
-                    {label: _('First'), value: 'first'},
-                    {label: _('Last'), value: 'last'},
-                    {label: _('None'), value: 'none'},
-                  ]}
-                />
-              </PlotlySection>
-              <PlotlySection name={_('Label Suffix')}>
-                <Dropdown
-                  label={_('Suffix')}
-                  attr={prefix + 'colorbar.ticksuffix'}
-                  options={[
-                    {label: _('C'), value: 'C'},
-                    {label: _('%'), value: '%'},
-                    {label: _('^'), value: '^'},
-                    {label: _('custom'), value: 'custom'},
-                  ]}
-                />
-                <Radio
-                  attr={prefix + 'colorbar.showticksuffix'}
-                  options={[
-                    {label: _('Every'), value: 'all'},
-                    {label: _('First'), value: 'first'},
-                    {label: _('Last'), value: 'last'},
-                    {label: _('None'), value: 'none'},
-                  ]}
-                />
-              </PlotlySection>
+              <DropdownCustom
+                label={_('Label Prefix')}
+                attr={prefix + 'colorbar.tickprefix'}
+                options={[
+                  {label: _('None'), value: ''},
+                  {label: _('x'), value: 'x'},
+                  {label: _('$'), value: '$'},
+                  {label: _('#'), value: '#'},
+                  {label: _('@'), value: '@'},
+                  {label: _('custom'), value: 'custom'},
+                ]}
+                customOpt="custom"
+                dafaultOpt=""
+                clearable={false}
+              />
+              <Radio
```

Comment:
thanks for removing the sections... please add a label here like I did in style/axes

Reply:
done